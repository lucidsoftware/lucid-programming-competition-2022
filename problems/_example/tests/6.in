iadedhdgqk, tittueyiux, lsjhsrnhen, vpqixdchak, tthmvhcpua, taoxrwezyf, qahyywnqbi, upzoosmryj, artbncvpgt, ygihyznvfl
9, 8, 3, 8, 9, 5, 8, 3, 9, 9
15
tcwvbvnprudzrfkaxvoh
lsjhsrnhen, ygihyznvfl, vpqixdchak, artbncvpgt, upzoosmryj, qahyywnqbi
7, 5, 7, 10, 8, 9
gnsenejvzcnkrreublfd
tthmvhcpua, tittueyiux, taoxrwezyf, upzoosmryj, iadedhdgqk
4, 6, 3, 3, 9
fxhugrgwugoqsjyktbpu
artbncvpgt, upzoosmryj, taoxrwezyf, qahyywnqbi, lsjhsrnhen, tthmvhcpua, iadedhdgqk, tittueyiux, vpqixdchak
4, 9, 1, 7, 9, 8, 2, 8, 4
lwuhibwilxnzvhleumcl
upzoosmryj, tthmvhcpua, tittueyiux, qahyywnqbi, vpqixdchak
8, 2, 10, 8, 9
ubtorbewocvczkubdhfe
artbncvpgt, vpqixdchak, upzoosmryj, qahyywnqbi, tthmvhcpua, taoxrwezyf, iadedhdgqk, ygihyznvfl, lsjhsrnhen
5, 9, 8, 7, 4, 1, 1, 10, 6
dpfeiiekpcaswjuhtubj
iadedhdgqk, qahyywnqbi
1, 6
yghsxvrppqyxspweuflr
upzoosmryj
2
fdpwsaicpwizqsxhwxtq
iadedhdgqk, ygihyznvfl, upzoosmryj, tittueyiux, lsjhsrnhen, tthmvhcpua, vpqixdchak, taoxrwezyf
4, 2, 1, 5, 7, 2, 4, 5
ganavofvcbwawijmuqev
taoxrwezyf, iadedhdgqk, vpqixdchak, qahyywnqbi, artbncvpgt, lsjhsrnhen, ygihyznvfl, upzoosmryj, tittueyiux
5, 6, 1, 9, 5, 6, 2, 6, 4
kiaoxsblbwelmduduuxg
lsjhsrnhen, ygihyznvfl, qahyywnqbi, upzoosmryj, taoxrwezyf
9, 4, 7, 9, 3
ekyvqfuqbbcgeumofjau
taoxrwezyf, vpqixdchak
2, 4
qlmqeobvwarigvbafknc
qahyywnqbi, iadedhdgqk, ygihyznvfl, upzoosmryj, tittueyiux, lsjhsrnhen, artbncvpgt, vpqixdchak
6, 10, 1, 4, 2, 9, 6, 4
knfwoiaxzkfxxhwsbdas
tthmvhcpua, tittueyiux, qahyywnqbi, artbncvpgt, ygihyznvfl, vpqixdchak, lsjhsrnhen, iadedhdgqk
8, 1, 10, 2, 1, 4, 9, 6
fngeuwjxojnalnfiquao
ygihyznvfl
4
cbmgcjuelocrondxiwlo
iadedhdgqk, tittueyiux, lsjhsrnhen, vpqixdchak, tthmvhcpua, taoxrwezyf, qahyywnqbi, upzoosmryj, artbncvpgt, ygihyznvfl
9, 8, 3, 8, 9, 5, 8, 3, 9, 9
