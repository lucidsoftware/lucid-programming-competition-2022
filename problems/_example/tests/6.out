cbmgcjuelocrondxiwlo
dpfeiiekpcaswjuhtubj
ekyvqfuqbbcgeumofjau
fngeuwjxojnalnfiquao
gnsenejvzcnkrreublfd
yghsxvrppqyxspweuflr
